aries_cloudagent/resolver/tests/test_base.py

```diff
@@ -20,6 +22,9 @@ async def setup(self, context):
     def supported_methods(self):
         return ["test"]
 
+    async def supports(self, profile: Profile, did: Union[str, DID]) -> bool:
+        return await super().supports(profile, did)
+
```

Comment:
As Luis pointed out, this isn't required. It will pass through by default if not declared.

Reply:
I was being explicit for example purpose, I can remove this?
please comment on pr #84